schema/bom-1.6.proto

```diff
@@ -1150,9 +1152,136 @@ message ModelCard {
       // With respect to the benefits and harms outlined, please describe any mitigation strategy implemented.
       optional string mitigationStrategy = 4;
     }
+    message EnvironmentalConsiderations {
+      // Describes energy consumption information incurred for one or more component lifecycle activities.
+      repeated EnergyConsumption energyConsumptions = 1;
```

Comment:
all the new messages and enums appear to be much centric to the existing message `ModelCardConsiderations`. especially `ActivityType` is ML-specific.
maybe nest those declarations in the message, instead of having them free floating somewhere.  like so: 
```
message Foo {
  message Bar { ... }
  enum Bazz { ... }
  message FooBar {
    repeated Foo foo = 1
  }

  optional Bar bar = 1
  FooBar foobar = 2
}
```

Reply:
@jkowalleck I was trying to consider which structures would be reusable independent of model card considerations (and perhaps, if time allowed, consider adding a means to declare them in the parent model card so that we have a means to allow these concepts to be used by others in v1.7 as Prabhu indicated interest in).

Reply:
I disagree. 

i will drop a PR to show what is possible.

Reply:
done